Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -328,6 +329,13 @@ def bump_formula_pr
       alias_rename.map! { |a| formula.tap.alias_dir/a }
     end
 
+    ohai "brew update-python-resources #{formula.name}"
+    unless args.dry_run? && !args.write?
+      # Ensure pipgrip is installed
+      odie '"pipgrip" must be installed (`brew install pipgrip`)' unless Formula["pipgrip"].any_version_installed?
+      PyPI.update_python_resources! formula, new_formula_version, silent: true, ignore_non_pypi_packages: true
```

Comment:
Should this check be in the method itself (given it's duplicated below)?

Reply:
Probably... Is it appropriate to add to the `PyPI` module as e.g. `ensure_pipgrip_installed`?